packages/components/addon/components/hds/tabs/index.js

```diff
@@ -69,7 +80,6 @@ export default class HdsTabsIndexComponent extends Component {
   @action
   onClick(tabIndex, event) {
     this.selectedTabIndex = tabIndex;
-    this.setTabIndicator(tabIndex);
```

Comment:
This no longer needs to manually set since the component is watching for changes to `this.selectedIndex` via the `did-update` modifier and will fire this method when it changes.